src/astgen

```diff
@@ -10,7 +10,7 @@ import sys
 
 
 class Node:
-    def __init__(self, name, superClass):
+    def __init__(self, name, superClass, file, line):
```

Comment:
```suggestion
    def __init__(self, name, superClass, file, lineno):
```

and change accessors etc.   Line I think of as a string, lineno the line number.